components/editor/overview.md

```diff
@@ -12,9 +12,9 @@ position: 0
 
 The <a href = "https://www.telerik.com/blazor-ui/editor" target="_blank">Blazor HTML Editor component</a> enables your users to create rich textual content through a What-You-See-Is-What-You-Get (WYSIWYG) interface and delivers a set of tools for creating, editing, and formatting text, paragraphs, lists, and other HTML elements.
```

Comment:
The Validation section should be split into two parts. The part about SignalR message size should be similar to https://docs.telerik.com/blazor-ui/components/fileselect/overview#large-file-support and have its own heading.